demo/simple/sysl-ints.sysl

```diff
@@ -0,0 +1,18 @@
+# sysl ints -o "sysl-ints.png" /sysl-ints --project Project -v
+# sysl ints -o "sysl-ints-epa.png" /sysl-ints --project Project --epa -v
+IntegratedSystem:
+    _1:
```

Comment:
_1 is confusing. IMHO might as well use this to give name to your intent.

Reply:
Well, the intent is to create a minimal sysl file for an integration diagram.

I used `_1` purposefully to demonstrate that it doesn't matter what you put here - it doesn't show up in the output. Do you have any other naming suggestions that make this point clear and are meaningful at the same time? Maybe something like `INTEGRATED_ENDPOINT_1` or `hidden_integrated_endpoint1` ?

Ideally IMO we wouldn't need `_1` or similar at all, but that option doesn't exist.

Reply:
Fair enough.

Reply:
I have actually changed it now to `integrated_endpoint_1` ...